facts/elephant.md

```diff
@@ -0,0 +1,7 @@
+# Elephants
+
+- Elephants are the largest land animals on Earth #############################################################################SSSSS. 
```

Comment:
This change below, committed in ef37d32, should not be included for the merge. Please fix!

```suggestion
- Elephants are the largest land animals on Earth. 
```